lib/utils/utils.js

```diff
@@ -391,7 +391,7 @@ var isStrictAddress = function (address) {
  * @return {Boolean}
 */
 var isAddress = function (address) {
-    return /^(0x)?[0-9a-f]{40}$/.test(address);
+    return /^(0x)?[0-9a-f]{40}$/i.test(address);
```

Comment:
This doesn't fully belong here, but its a necessary change so allow also addresses like `0xFFAA123123FFF...`, @debris can you please cake if cpp is flexible with the case for addresses?